http-python3.12-django5.0/README.md

```diff
@@ -0,0 +1,19 @@
+# Python Django web application
+
+To run this example, first [install the `kraft` CLI tool](https://unikraft.org/docs/cli).
+
+Then, clone this repository and `cd` into this directory.
+To deploy this application on KraftCloud, invoke:
+
+```
+kraft cloud deploy -M 512 -p 443:80 .
```

Comment:
Can we switch this to `8080`, most other examples/catalog entries are using this (we should switch those that aren't), and `8080` helps with local development.

Reply:
I really dislike this whole port mess. Port 80 is the standard HTTP port...

Reply:
Also for development you would use the `manage.py runserver` command.

Reply:
We should just use `8080`. `80` is a restricted port on some hosts which require admin privileges and `8080` is common for local web development (along with `3000`).

Reply:
This limitation does not apply to docker or kraftcloud